meta-balena-zeus/recipes-containers/tini/tini_0.14.0.bbappend

```diff
@@ -0,0 +1,2 @@
+# Thud static PIE seems broken as binary gets into segfault at runtime
+SECURITY_CFLAGS_pn-${PN} += "${SECURITY_NOPIE_CFLAGS}"
```

Comment:
Is this still the case with zeus?

Reply:
pulled tini into meta-balena-common and removed this patch